deeprank/features/FullPSSM.py

```diff
@@ -218,6 +219,10 @@ def get_feature_value(self, cutoff=5.5):
         # get feature values
         for res in ctc_res_with_pssm:
             chain = {'A': 0, 'B': 1}[res[0]]
+            print(res)
+            if res == ('B', 47, 'THR'):
+                print(res)
```

Comment:
I don't know what that does but if it's important could we replace the print statement with a warnings.warn('fdsfdfds') to precisely describe what the issue is ?

Reply:
Sorry, all those print statements are for debugging the missing atom problem. I forgot to delete them. Now deleted.